module-4/backend/week_three.md

```diff
@@ -5,15 +5,48 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+
+An open source server written in JavaScript.
+
 2. What is Express? What is Express similar to in the Ruby world?
+
+Express is a lightweight web framework similar to Sinatra.
+
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+
+```
+app.get('<endpoint>', function(request, response) {
+  <controller>.<controller function>(request, response)
+})
+```
+
```

Comment:
Which could be refactored as:

```
app.get(<endpoint>, <controller.controllerfunction>)
```

request and response will be implicitly passed as arguments to the callback function